code/game/objects/items/apc_frame.dm

```diff
@@ -6,8 +6,8 @@
 	item_state = "syringe_kit"
 	w_class = WEIGHT_CLASS_SMALL
 	var/result_path
-	var/inverse = 0
-	// For inverse dir frames like light fixtures.
+	var/inverse = 0 // For inverse dir frames like light fixtures.
+	var/onto_wall = FALSE //If the object places onto the wall instead of the floor
```

Comment:
They are pixel-shifted, but I'm not really sure how to pixel-shift the new intercoms effectively based on direction. I guess I could do a switch() and check for the cardinals.